src/test/regress/expected/non_super_user_object_metadata.out

```diff
@@ -0,0 +1,303 @@
+SET citus.enable_ddl_propagation TO OFF;
+CREATE SCHEMA local_schema;
+SET search_path TO local_schema;
+-- Create type and function that depends on it
+CREATE TYPE test_type AS (f1 int, f2 text);
+CREATE FUNCTION test_function(int) RETURNS test_type
+    AS $$ SELECT $1, CAST($1 AS text) || ' is text' $$
+    LANGUAGE SQL;
+-- Create various objects
+CREATE TYPE mood AS ENUM ('sad', 'ok', 'happy');
+CREATE SEQUENCE test_sequence;
+-- show that none of the objects above are marked as distributed
+SELECT pg_identify_object_as_address(classid, objid, objsubid) from citus.pg_dist_object where objid = 'local_schema'::regnamespace::oid;
+ pg_identify_object_as_address 
+-------------------------------
+(0 rows)
+
+SELECT pg_identify_object_as_address(classid, objid, objsubid) from citus.pg_dist_object where objid = 'local_schema.mood'::regtype::oid;
+ pg_identify_object_as_address 
+-------------------------------
+(0 rows)
+
+SELECT pg_identify_object_as_address(classid, objid, objsubid) from citus.pg_dist_object where objid = 'local_schema.test_type'::regtype::oid;
+ pg_identify_object_as_address 
+-------------------------------
+(0 rows)
+
+SELECT pg_identify_object_as_address(classid, objid, objsubid) from citus.pg_dist_object where objid = 'local_schema.test_sequence'::regclass::oid;
+ pg_identify_object_as_address 
+-------------------------------
+(0 rows)
+
+SELECT pg_identify_object_as_address(classid, objid, objsubid) from citus.pg_dist_object where objid = 'local_schema.test_function'::regproc::oid;
+ pg_identify_object_as_address 
+-------------------------------
+(0 rows)
+
+SET client_min_messages TO ERROR;
+CREATE USER test_user;
+SELECT run_command_on_workers($$CREATE USER test_user;$$);
```

Comment:
minor: To make this test pass on enterprise, you may need:
```SQL
SELECT 1 FROM run_command_on_workers($$CREATE USER test_user;$$);
```

so that the outputs are the same